Description: Use qtchooser to find moc
--- gcin-2.8.3+dfsg1.orig/configure
+++ gcin-2.8.3+dfsg1/configure
@@ -314,13 +314,7 @@ if [ $QT4_IM = 'N' ]; then
   mv gcin.spec gcin.spec.tmp
   grep -v qt4/plugins/inputmethods gcin.spec.tmp > gcin.spec
 else
-  for i in /usr/share/qt4/bin/moc /usr/lib/qt4/bin/moc /usr/lib64/qt4/bin/moc /usr/bin/moc
-  do
-    [ ! -x $i ] && continue
-    QT4_MOC=$i
-    echo "QT4_MOC=$QT4_MOC"
-    break
-  done
+  QT4_MOC="qtchooser -run-tool=moc -qt=4"
 fi
 
 
@@ -328,13 +322,7 @@ if [ $QT5_IM = 'N' ]; then
   mv gcin.spec gcin.spec.tmp
   grep -v qt5/plugins/inputmethods gcin.spec.tmp > gcin.spec
 else
-  for i in /usr/share/qt5/bin/moc /usr/lib/qt5/bin/moc /usr/lib/x86_64-linux-gnu/qt5/bin/moc /usr/lib64/qt5/bin/moc /usr/bin/moc
-  do
-    [ ! -x $i ] && continue
-    QT5_MOC=$i
-    echo "QT5_MOC=$QT5_MOC"
-    break
-  done
+  QT5_MOC="qtchooser -run-tool=moc -qt=5"
 fi
 
 
